01_fib/CameronSchafer/fib.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+//base fibonacci function
+func fib(n int) {
+	sign, n1, n2 := 1, 0, 1
+	//check if use nega or normal fib sequence
+	if n < 0 {
+		sign = -1
+	}
+
+	n *= sign
+	//loop until all numbers have been printed
+	for i := 0; i < n; i++ {
```

Comment:
minor suggestion: count from 1 to n, instead of 0 to n-1 (i.e. `for i := 1; i <= n; i++`) because we are computing the nth fibonacci number, not the nth-1 fibonacci number.